conda/base/constants.py

```diff
@@ -90,6 +90,8 @@
 
 UNKNOWN_CHANNEL = "<unknown>"
 
+PIP_PSEUDOCHANNEL = "<pip>"
```

Comment:
(Though to be clear: this doesn't inject the `<pip>` pseudochannel into normal operations _yet_. It's just there for use with`egg_info`; e.g. `conda list`.)